bookstore_api/serializers.py

```diff
@@ -11,4 +11,6 @@ class Meta:
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = UserProfiles
-        fields = '__all__'
\ No newline at end of file
+        fields = '__all__'
```

Comment:
Same thing with this one. Serializers.py - nothing is being added to it and I have a total update on my merge. this file should not be updated